libsolutil/pch.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Also, `pch.h` is not a very clear name for a header file. Is that meant to be a "precompiled header" or something? We already have some headers for common stuff (names starting with `Common`) but this is very specific and should have a descriptive name, e.g. `MethodCheck.h`.